src/components/itemContextMenu.js

```diff
@@ -589,9 +614,12 @@ function play(item, resume, queue, queueNext) {
             serverId: item.ServerId
         });
     } else {
+        const sortParentId = item.IsFolder ? ('items-' + item.Id) : libraryMenu.getTopParentId() + '-' + getSettingsKey(item);
+        const sortValues = userSettings.getSortValues(sortParentId);
         playbackManager[method]({
             items: [item],
-            startPositionTicks: startPosition
+            startPositionTicks: startPosition,
+            queryOptions: sortValues
```

Comment:
```suggestion
            queryOptions: {
                 SortBy: sortValues.sortBy, 
                 SortOrder: sortValues.sortOrder
              }
```

Reply:
It could be a `PhotoAlbum`. The idea is to get its sort options (if any), as if you went into it first and then click `play all`.

Reply:
play all executed in shortcuts.js at line 125

Reply:
Removed.

Thank you for the help :+1:

Reply:
To test what I was talking about:
1. Go to some photo album.
2. Change its sort order - needs to be noticeable.
3. Go one level up.
4. Click `...` button over that photo album.
5. Click `Play` from the menu.
6. It plays in default order (`SortName Ascending`). _After reverting changes._

The question is whether we want it to play in the default order or in a customized order.

Reply:
Well. It does save in the localsettings it's last sort order for that Album so I don't know. 

Either way it doesn't seem to be like it's bad behavior. One could argue that if you wanted to play without the default order you should go to the photo album itself?

Let me know what yall think.